playbooks/openshift-etcd/private/scaleup.yml

```diff
@@ -85,3 +85,14 @@
   - import_role:
       name: openshift_master
       tasks_from: update_etcd_client_urls.yml
+
+- name: Update Service Catalog API server with new etcd members
```

Comment:
This playbook would scale up the etcd cluster, used by Kubernetes and Openshift.

Service Catalog starts another etcd cluster for it needs, so `openshift-etcd` playbook should not affect service catalog etcd cluster. This needs to be moved to a different playbook

Reply:
I might have gotten the concept wrong then. My idea was the following: upon installation of service catalog, an apiserver daemonset is created from the j2 template where container arguments are set. Here --etcd-servers gets openshift_master_etcd_urls values which in my understanding was reference to existing etcd members. If we run the etcd scaleup playbook, this argument is not updated. The pull request was to update the apiserver ds where the new etcd member url-s are added to the --etcd-server argument.